packages/web/src/js/__tests__/Accordion.test.ts

```diff
@@ -0,0 +1,152 @@
+import { clearFixture, getFixture } from '../../../tests/helpers/fixture';
```

Comment:
Why is here this test? There is no Accordion JS plugin to test. I suppose that the content of this file should be inside the Collapse test.

Reply:
Ok, so I've added this Accordion part to Collapse.test and I'm testing the data-parent and its binding ie. I have the second one open, I click on the first one and then check that the first one is open and the second one is closed.